templates/CONTRIBUTING.md

```diff
@@ -0,0 +1,132 @@
+3. **You know about the [MeiliSearch community](https://docs.meilisearch.com/resources/contact.html). Please use this for help.**
+
+## How to Contribute
+
+1. Ensure your change has an issue! Find an [existing issue](https://github.com/meilisearch/XXX/issues/) or [open a new issue](https://github.com/meilisearch/XXX/issues/new). This is where you can get a feel if the change will be accepted or not.
+2. Once approved, [fork the XXX repository](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) in your own GitHub account.
+3. [Create a new Git branch](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-and-deleting-branches-within-your-repository).
+4. Review the [Development Workflow](#workflow) section that describes the steps to maintain the repository.
+5. Make your changes.
+6. [Submit the branch as a PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork) pointing to the `master` branch of the main XXX repo. A team member should comment and/or review your Pull Request with a few days. Although, depending on the circumstances, it may take longer.<br>
+   About the PR title, we do not adopt any type of  naming style, but **please use something descriptive of your changes**. The title of the PR will be indeed automatically added to the next [release changlogs](https://github.com/meilisearch/XXX/releases/).
+
+## Development Workflow
+
+---
+
+COMMENT TO REMOVE
+
+This section should be filled according to the repository and should contain:
+- A "Setup" section (if needed) with the command to install the dependencies for the contributor (e.g.: `yarn --dev`).
+- A "Tests and Linter" section with the command to run the tests, the linter and the linter-autofix (if possible).
+- A "Release Process" section that describes the process before publishing a new release.
+- Other useful tips/steps for the contributor (e.g.: how to build, how to debug).
+
+/!\ The following lines are just a template example and should be completed with right names and right commands
+
+---
+
+### Setup
+
+```bash
+$ <Add the right command here>
+```
+
+### Tests and Linter
+
+Each PR should pass the tests and the linter to be accepted.
+
+```bash
+# Tests
+$ <Add Docker command to run MeiliSearch here>
+$ <Add the right command here>
+# Linter
+$ <Add the right command here>
+# Linter with fixing
+$ <Add the right command here>
+```
+
+### Release Process
+
+MeiliSearch tools follow the [Semantic Versioning Convention](https://semver.org/).
+
+
+#### Automatiazed Changelogs
+
+For each PR merged on `master`, a GitHub Action is running and updates the next release description as a draft release in the [GitHub interface](https://github.com/meilisearch/XXX/releases). If you don't have the permission to this repository, you will not be able to see the draft release until the release will be published.
```

Comment:
```suggestion
For each PR merged on `master`, a GitHub Action is running and updates the next release description as a draft release in the [GitHub interface](https://github.com/meilisearch/XXX/releases). If you don't have the permission to this repository, you will not be able to see the draft release until the release is published.
```

Reply:
For each PR merged on `master`, a GitHub Action is running and updates the next release description as a draft release in the [GitHub interface](https://github.com/meilisearch/XXX/releases). If you don't have the right access to this repository, you will not be able to see the draft release until the release will be published.

Reply:
Maybe `write access`?